module1/build tools/api/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'java'
+}
+
+group 'com.epam'
+version '1.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile files('C:\\Users\\kirill\\Desktop\\lab\\module1\\build tools\\jar\\core.jar')
```

Comment:
According to the task, you need only one prepared jar: "utils-1.3.5.jar". You need to define dependency from core-module, not from core.jar

Reply:
Added dependency from core-module. Deleted dependency from core.jar.